src/components/ReadTheDocsInfo.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const ReadTheDocsInfo = (props: { docsUrl: string }): JSX.Element => {
+    const { docsUrl } = props;
+    return (
+        <div className="card alert alert-info" role="alert">
+            <div className="card-body">
+                <i className="fa-solid fa-circle-info fa-2xl me-2"></i>
+
+                <a href={docsUrl} target="_blank" rel="noreferrer" className="alert-link align-middle">
+                    Read about this in the documentation…
```

Comment:
this calls for translation

Reply:
https://github.com/nurikk/zigbee2mqtt-frontend/pull/2020/commits/0eb757b162caf99b85d638a178f9af53852c9670